spec/adapters/alma_adapter_spec.rb

```diff
@@ -311,33 +313,38 @@
     it "reports CDL when available" do
       availability = adapter.get_availability_one(id: "9965126093506421")
       holding = availability["9965126093506421"]["22202918790006421"]
-      expect(holding[:holding_type]).to eq "physical"
       expect(holding[:status_label]).to eq "Unavailable"
       expect(holding[:cdl]).to eq true
     end
 
     it "reports some items available" do
       availability = adapter.get_availability_one(id: "9921799253506421")
       holding = availability["9921799253506421"]["22201236200006421"]
-      expect(holding[:holding_type]).to eq "physical"
       expect(holding[:status_label]).to eq "Some items not available"
     end
 
-    it "reports availability of portfolios" do
+    it "ignores electronic resources" do
       availability = adapter.get_availability_one(id: "99122426947506421")
-      portfolio = availability["99122426947506421"]["53469873890006421"]
-      expect(portfolio[:holding_type]).to eq "portfolio"
-      expect(portfolio[:status_label]).to eq "Available"
+      empty_availability = { "99122426947506421" => {} }
+      expect(availability).to eq empty_availability
     end
 
-    it "reports availability for other" do
-      # This kind of resource is new in Alma and still needs some work,
-      # but for now we at least test that we are executing the branch of
-      # code that handles them.
+    it "reports availability (without holding_id) for items in temporary locations" do
```

Comment:
This test and the one below it are the only relevant test for handling temporary locations. The rest of the changes in the tests are to account for the slightly different response that we returning now that we are only reporting on physical items (i.e. ignoring electronic portfolios and digital items)